implementations/go/.env

```diff
@@ -0,0 +1,3 @@
+{
+    "token":"EAAEDNuZAnTygBAJEOqZCyxUsEGmq3HW0XEnleiZCRsFp7VPbvLsZCRa3GRsGe2AabSdwdbgt4dmElZBK2gay0uPpkhgDZCwvWcUEcT6COUChSevNZAhWFWJSBKBlA8E7LwN8dK1nTStfCtkXXrvwU12WBYWKwqIiZABz0RK6CEPvhAZDZD"
+}
```

Comment:
🙈You'll need to add to add this to your .gitignore

Reply:
Its been removed already, and an implementation added...